source/postcard-schema/src/impls/builtins_std.rs

```diff
@@ -21,6 +21,14 @@ impl Schema for std::string::String {
     };
 }
 
+#[cfg_attr(docsrs, doc(cfg(any(feature = "alloc", feature = "use-std"))))]
```

Comment:
`PathBuf` doesn't exist in `alloc` so this should probably be `#[cfg_attr(docsrs, doc(cfg(feature = "use-std")))]`

Reply:
Yeah, @bugadani it would also be good to copy the impl over to the `builtins_alloc` item as well.

Reply:
Sorry for the noise, I got a bit confused how this all works. I can't get `builtins_alloc` to display locally at all, so I'm not sure what I did is correct.